.github/workflows/automerge-workflow.yml

```diff
@@ -0,0 +1,37 @@
+jobs:
+  automerge:
+    name: Automerge Workflow
+    runs-on: ubuntu-latest
+    steps:
+      - env:
+          GITHUB_TOKEN: ${{ secrets.PULUMI_BOT_TOKEN }}
+          MERGE_COMMIT_MESSAGE: pull-request-title
+          MERGE_FORKS: "false"
+          MERGE_LABELS: automation/merge
```

Comment:
Just to check my understanding: It's the presence of this label on _any_ PR that will result in an auto-merge on successful completion of the check suite. Correct? And this label only gets applied (automatically) with `build-tfgen-provider-docs` and `build-pulumi-cli-docs`?

Reply:
Correct - this means that the use case can be extended where required, but the only place that adds it by default is the provider and cli docs PRs

Reply:
We can add any tags in here that you want, but it still allows people in our org to get around the restrictions - this is a process / cultural problem if it happens right?

Reply:
Ok, thanks for confirming. cc @leezen just as a head's up. I'm okay to try it as long as no objections from others.